quantum-api/assets/languages/quantum-api-javascript.css

```diff
@@ -198,6 +198,10 @@
 }
 
 /* Changelog */
+.qm-changelog-javascript-function + .qm-changelog-javascript-arg:before,
+.qm-changelog-javascript-constructor + .qm-changelog-javascript-arg:before,
+.qm-changelog-javascript-method + .qm-changelog-javascript-arg:before,
+.qm-changelog-javascript-arg + .qm-changelog-javascript-property:before,
```

Comment:
Related to #144 changes

This is so the headers render nicely with a `.` between constructor + arguments